pages/Configuring/Binds.md

```diff
@@ -166,9 +166,10 @@ l -> locked, will also work when an input inhibitor (e.g. a lockscreen) is activ
 r -> release, will trigger on release of a key.
 e -> repeat, will repeat when held.
 n -> non-consuming, key/mouse events will be passed to the active window in addition to triggering the dispatcher.
-m -> mouse, see below
+m -> mouse, see below.
 t -> transparent, cannot be shadowed by other binds.
 i -> ignore mods, will ignore modifiers.
+s -> separate, will arbitrarily combine keys between each mod/key, see [Keysym combos](./Binds.md/#keysym-combos) above.
```

Comment:
```suggestion
s -> separate, will arbitrarily combine keys between each mod/key, see [Keysym combos](#keysym-combos) above.
```

`#` will find the id in the current document.

Reply:
My bad, thx!